bin/plugin/group-aclkeeper/groupSetServers

```diff
@@ -0,0 +1,190 @@
+#! /usr/bin/env perl
+# vim: set filetype=perl ts=4 sw=4 sts=4 et:
+use common::sense;
+use JSON;
+
+use File::Basename;
+use lib dirname(__FILE__) . '/../../../lib/perl';
+use OVH::Result;
+use OVH::Bastion;
+use OVH::Bastion::Plugin qw( :DEFAULT help );
+use OVH::Bastion::Plugin::ACL;
+
+my $remainingOptions = OVH::Bastion::Plugin::begin(
+    argv               => \@ARGV,
+    header             => "replace a group's current ACL by a new one",
+    userAllowWildcards => 1,
+    options            => {
+        "group=s"     => \my $group,
+        "dry-run"     => \my $dryRun,
+        "skip-errors" => \my $skipErrors,
+    },
+    helptext => <<'EOF',
+Replace a group's current ACL by a new list
+
+Usage: --osh SCRIPT_NAME --group GROUP [OPTIONS]
+
+  --group GROUP  Specify which group this machine should be added to
+  --dry-run      Don't actually modify the ACL, just report whether the input contains errors
+  --skip-errors  Don't abort on STDIN parsing errors, just skip the non-parseable lines
+
+The list of the assets to constitute the new ACL should then be given on STDIN,
+in the following format: ``[USER@]HOST[:PORT][ COMMENT]``, with USER and PORT being optional,
+and HOST being either a hostname, an IP, or an IP block in CIDR notation. The COMMENT is
+also optional, and may contain spaces.
+
+Examples::
```

Comment:
```suggestion
Examples:
```

Reply:
It's actually an `rst` trick to have the next lines put into a block for the auto-generated documentation ;)